client/src/utils/export-util.js

```diff
@@ -7,4 +9,30 @@ import { parse as toCSV } from 'json2csv';
  */
 export function exportToCsv(fields, data) {
     return 'data:text/csv;charset=utf-8,' + toCSV(data, {fields});
-}
\ No newline at end of file
+}
+
+/**
+ * Export the specified data with the specified fields to CSV format
+ * @param fields - Array of the names of the columns for the CSV
+ * @param data - Array of data representing the rows of the CSV
+ */
+export function exportToPdf(fields, data, exportTitle) {
+    const dataArr = []
+    for (const field in data[0]) {
+        dataArr.push(data[0][field])
```

Comment:
As mentioned before with Dr. Decker, width of columns causing weird text wrapping issues. I believe this is because of the tags column. If there is more than one tag, it will separate by commas but no space after the comma. I think the lack of space between the comma and next tag causes the spacing issues.

Reply:
good catch, I will look into fixing this

Reply:
added spaces in most recent commit